Opened PR on testPluginActive

NDR0216 opened the pull request:
I opened https://github.com/AmadeusITGroup/jenkins-opentracing-plugin/pull/8 to fix testPluginActive.

More discussion can be found on campuswire#706.

NDR0216 commented:
> Why did you open the real PR from your `master` branch? How can removing that one import help?

So should I create a new branch when I commit the fix?

For the second question, that import was removed after I ran `mvn com.coveo:fmt-maven-plugin:format`.

NDR0216 commented:
OK. I would rename the branch.

> It doesn't matter how you made an unnecessary change, just don't make such changes.
I did not get the point of why it is unnecessary. If I did not change it, the project could not be built. Why shouldn't I change it?

darko-marinov commented:
Are you saying that the project didn't compile (i.e., `javac` failed) without removing that import, or that some else failed (maybe some Maven plugin for checking style or unused imports)?

NDR0216 commented:
> Are you saying that the project didn't compile (i.e., `javac` failed) without removing that import, or that some else failed (maybe some Maven plugin for checking style or unused imports)?

It's the latter case, some Maven plugin for checking style or unused imports. If it is unnecessary, I can remove it.

darko-marinov commented:
Remove that change; based on your description, it's unnecessary for the code to compile.

NDR0216 commented:
> Remove that change; based on your description, it's unnecessary for the code to compile.

I see. I got the point.
I removed that part. and updated the PR.

darko-marinov commented:
Please squash your 2 commits into 1, without closing the PR.

NDR0216 commented:
> Please squash your 2 commits into 1, without closing the PR.

I squashed them. Please check.
